modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -442,30 +442,28 @@
                         "example": "Compensation"
                       },
                       "claimDate": {
-                        "type": [
-                          "string",
-                          "null"
-                        ],
                         "format": "date",
+                        "type": "date",
```

Comment:
sorry, this was a lack of research for the ticket / wording issue - looked in to the date formatting for OAS and these are actually a type of string, so we do have an update needed here but not to much of one. See the two links below.
https://swagger.io/docs/specification/data-models/data-types/#string

Note that there is no null type; instead, the [nullable](https://swagger.io/docs/specification/data-models/data-types/#null) attribute is used as a modifier of the base type.